frontend_lib/src/component/Button/IconButton.styl

```
@@ -41,10 +40,17 @@
   &__link
     background-color transparent
     border none
-    text-decoration underline
-    padding 0 5px
+    padding 0
     &:hover
       text-decoration underline
+    &_light
+      color mediumDarkGrey
+      &:hover
+        color darkGrey
+    &_dark
+      color fontColor
+      &:hover
+        color black
```

in the BOM norm, each "children" should be separated by 2 underscores.
If _light and _dark and modifiers of the class, then the norm expect a dash as a separation char
```suggestion
    &-light
      color mediumDarkGrey
      &:hover
        color darkGrey
    &-dark
      color fontColor
      &:hover
        color black
```

---

Updated.